inc/saitam.h

```diff
@@ -1012,48 +481,11 @@ typedef enum _sai_tam_tel_type_attr_t
     /**
      * @brief Flow - Flow ID
      *
-     * @type sai_uint32_t
-     * @flags MANDATORY_ON_CREATE | CREATE_AND_SET
-     * @condition SAI_TAM_TEL_TYPE_ATTR_TAM_TELEMETRY_TYPE == SAI_TAM_TELEMETRY_TYPE_FLOW
+     * type sai_uint32_t
+     * flags MANDATORY_ON_CREATE | CREATE_AND_SET
+     * condition SAI_TAM_TEL_TYPE_ATTR_TAM_TELEMETRY_TYPE == SAI_TAM_TELEMETRY_TYPE_FLOW
+     * SAI_TAM_TEL_TYPE_ATTR_FLOW_ID,
      */
```

Comment:
It looks like some dangling text remains after removing SAI_TAM_TEL_TYPE_ATTR_FLOW_ID